homeassistant/components/fritzbox_callmonitor/strings.json

```diff
@@ -37,5 +37,18 @@
     "error": {
       "malformed_prefixes": "Prefixes are malformed, please check their format."
     }
+  },
+  "entity": {
+    "sensor": {
+      "fritzbox_callmonitor": {
+        "name": "Call monitor",
+        "state": {
+          "ringing": "ringing",
```

Comment:
The States should also move to the separate PR. Until there is a solution for the phone book name, you can create a PR to make the sensor an Enum with these values.

Reply:
See https://github.com/home-assistant/core/pull/99762